coconut/control/control.go

```diff
@@ -420,26 +420,17 @@ func DestroyEnvironment(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.
 	}
 	envId := args[0]
 
-	// Check current state first
-	var envResponse *pb.GetEnvironmentReply
-	envResponse, err = rpc.GetEnvironment(cxt, &pb.GetEnvironmentRequest{Id: envId}, grpc.EmptyCallOption{})
+	keepTasks, err := cmd.Flags().GetBool("keep-tasks")
 	if err != nil {
-		return
-	}
-
-	allowedState := "CONFIGURED"
-	if envResponse.GetEnvironment().GetState() != allowedState {
-		fmt.Fprint(o, "cannot teardown environment\n")
-		fmt.Fprintf(o, "teardown is allowed in state %s, but environment %s is in state %s\n", allowedState, envId, envResponse.GetEnvironment().GetState())
-		return
```

Comment:
How does this affect the quality of the error text provided by `coconut` when the environment cannot be torn down due to a disallowed state?

Reply:
`DestroyEnvironment` from server.go will send back with an error with a similar message if the environment cannot be destroyed due to the current state:
""cannot destroy environment in state %s"